### DISCUSSION TITLE
fsleyes -> Unable to set default locale: 'unsupported locale setting'

### DISCUSSION THREAD

**Discussion by bvernay** [neuropoly/axondeepseg#495] [Help needed] [answered]
Good morning,

First of all thank you for making your tool available to the research community, it is much appreciated.

I have followed the installation instructions (using anconda not miniconda). The quick and comprehensive checks do pass and the notebooks are working.

The issue is with starting the GUI as I get the following error and it hangs on the loading image
input = fsleyes
output = 10:53:26: Erreur : Unable to set default locale: 'unsupported locale setting'

Any suggestion where the issue might be? 

Many thanks

********************************************
Bertrand Vernay, PhD
Head of the Photonic Microscopy Facility
http://quest.igbmc.fr/
http://www.igbmc.fr/technologies/5/team/55/
https://github.com/bvernay
Office:+33 (0)3 69 48 51 27
Office: CBI, E1009
ORCID 0000-0002-7843-3872

IGBMC - CNRS UMR 7104 - Inserm U 1258
1 rue Laurent Fries
BP 10142
67404 Illkirch CEDEX
France Tél +33 (0)3 88 65 32 00
Fax +33 (0)3 88 65 32 01

**Comment by mathieuboudreau** [comment, marked as answer]
Hi @bvernay !

Could you please run `pip freeze`, `conda list`, and `which fsleyes` and copy-paste the output of those commands here? It'll let us know more about the environment you're running before we can help you out.

**Reply by MicPhotonIGBMC** [reply]
Hi Mathieu

pip freeze
OUTPUT =
ERROR: Error [WinError 2] Le fichier spécifié est introuvable while executing command git config --get-regexp 'remote\..*\.url'                                   WARNING: cannot determine version of editable source in c:\users\vernayb\documents\axondeepseg (git command not found in path)                                    absl-py @ file:///home/conda/feedstock_root/build_artifacts/absl-py_1615404881292/work                                                                            alabaster==0.7.12                                                                                                                                                 albumentations==0.3.0                                                                                                                                             argon2-cffi @ file:///D:/bld/argon2-cffi_1605217174718/work                                                                                                       astor @ file:///home/conda/feedstock_root/build_artifacts/astor_1593610464257/work                                                                                async-generator==1.10                                                                                                                                             atomicwrites @ file:///home/conda/feedstock_root/build_artifacts/atomicwrites_1588182545583/work                                                                  attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1605083924122/work                                                                                -e AxonDeepSeg==3.2.0                                                                                                                                             Babel @ file:///home/conda/feedstock_root/build_artifacts/babel_1605182336601/work                                                                                backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work                                                                          backports.functools-lru-cache @ file:///home/conda/feedstock_root/build_artifacts/backports.functools_lru_cache_1616843134209/work                                bleach @ file:///home/conda/feedstock_root/build_artifacts/bleach_1612213472466/work                                                                              brotlipy==0.7.0                                                                                                                                                   certifi==2020.12.5                                                                                                                                                cffi @ file:///D:/bld/cffi_1613414034267/work                                                                           chardet @ file:///D:/bld/chardet_1610093644405/work                                                                                                                                                         cloudpickle @ file:///home/conda/feedstock_root/build_artifacts/cloudpickle_1598400192773/work                                                                    colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1602866480661/work                                                                          contextlib2==0.6.0.post1                                                                                                                                          coverage @ file:///D:/bld/coverage_1614551457425/work                                                                                                             cryptography @ file:///D:/bld/cryptography_1616851687511/work                                                           cycler==0.10.0                            cytoolz==0.11.0                                                                                                                                                   dask @ file:///home/conda/feedstock_root/build_artifacts/dask-core_1616804774713/work                                                                             decorator==4.4.2                                                                                                        defusedxml @ file:///home/conda/feedstock_root/build_artifacts/defusedxml_1615232257335/work                                                                                                                docutils @ file:///D:/bld/docutils_1610127977236/work                                                                                                             entrypoints @ file:///home/conda/feedstock_root/build_artifacts/entrypoints_1605121927639/work/dist/entrypoints-0.3-py2.py3-none-any.whl                          et-xmlfile==1.0.1                                                                                                                                                 fsleyes @ file:///D:/bld/fsleyes_1591738230381/work                                                                                                               fsleyes-props @ file:///home/conda/feedstock_root/build_artifacts/fsleyes-props_1616949869693/work                      fsleyes-widgets @ file:///home/conda/feedstock_root/build_artifacts/fsleyes-widgets_1616889336086/work                                                                                                      fslpy @ file:///home/conda/feedstock_root/build_artifacts/fslpy_1612895308155/work                                                                                fsspec @ file:///home/conda/feedstock_root/build_artifacts/fsspec_1614107550004/work                                    gast @ file:///home/conda/feedstock_root/build_artifacts/gast_1596839682936/work                                        grpcio @ file:///D:/bld/grpcio_1614828548869/work                                   h5py @ file:///D:/bld/h5py_1602551956383/work                                                                           idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1593328102638/work                                                                                                                            imageio @ file:///home/conda/feedstock_root/build_artifacts/imageio_1594044661732/work                                  imagesize==1.2.0                                                                                                        imgaug==0.4.0                                                                       importlib-metadata @ file:///D:/bld/importlib-metadata_1617068693035/work                                               indexed-gzip @ file:///D:/bld/indexed_gzip_1616518039918/work                                                           iniconfig @ file:///home/conda/feedstock_root/build_artifacts/iniconfig_1603384189793/work                                                                                                                                                            ipykernel @ file:///D:/bld/ipykernel_1613853146220/work/dist/ipykernel-5.5.0-py3-none-any.whl                           ipython @ file:///D:/bld/ipython_1616837008098/work                                                                     ipython-genutils==0.2.0                                                                                                 ipywidgets @ file:///home/conda/feedstock_root/build_artifacts/ipywidgets_1609995587151/work                                  isodate==0.6.0                                                                                                                                                    jdcal==1.4.1                                                                                                                                                      jedi @ file:///D:/bld/jedi_1610146981085/work                                                                           Jinja2 @ file:///home/conda/feedstock_root/build_artifacts/jinja2_1612119311452/work                                                                                                                        jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema_1614815863336/work                                                                      jupyter @ file:///D:/bld/jupyter_1611872076438/work                                                                                                               jupyter-client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1615693636836/work                    jupyter-console @ file:///home/conda/feedstock_root/build_artifacts/jupyter_console_1616560109969/work                                                                                                      jupyter-core @ file:///D:/bld/jupyter_core_1612125436808/work                                                           jupyterlab-pygments @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_pygments_1601375948261/work                                                                                              jupyterlab-widgets @ file:///home/conda/feedstock_root/build_artifacts/jupyterlab_widgets_1609173350931/work            Keras==2.3.1                              Keras-Applications==1.0.8                                                                                                                                         Keras-Preprocessing @ file:///home/conda/feedstock_root/build_artifacts/keras-preprocessing_1610713559828/work          kiwisolver @ file:///D:/bld/kiwisolver_1610099949969/work                                                                                                                                                   locket==0.2.0                                                                                                           Mako @ file:///home/conda/feedstock_root/build_artifacts/mako_1610659158978/work                                                                                                                            Markdown @ file:///home/conda/feedstock_root/build_artifacts/markdown_1614595805172/work                                                                          MarkupSafe @ file:///D:/bld/markupsafe_1610127696479/work                                                               matplotlib @ file:///D:/bld/matplotlib-suite_1611858873832/work                                                                                                                                             mistune @ file:///D:/bld/mistune_1610113018274/work                                                                                                               more-itertools @ file:///home/conda/feedstock_root/build_artifacts/more-itertools_1612756563342/work                    mpld3 @ file:///home/conda/feedstock_root/build_artifacts/mpld3_1606516286627/work                                                                                                                          nbclient @ file:///home/conda/feedstock_root/build_artifacts/nbclient_1614336084111/work                                nbconvert @ file:///D:/bld/nbconvert_1605401997174/work                                                                 nbformat @ file:///home/conda/feedstock_root/build_artifacts/nbformat_1611005694671/work                                                                                                                                                              nest-asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1605195931949/work                        networkx @ file:///home/conda/feedstock_root/build_artifacts/networkx_1598210780226/work                                                                                                                    nibabel @ file:///home/conda/feedstock_root/build_artifacts/nibabel_1606620997676/work                                                                            notebook @ file:///D:/bld/notebook_1616419450136/work                                                                   numpy @ file:///D:/bld/numpy_1616933410625/work                                                                                                                                                             olefile @ file:///home/conda/feedstock_root/build_artifacts/olefile_1602866521163/work                                                                            opencv-contrib-python==4.5.1.48                                                                                         opencv-python-headless==4.5.1.48          openpyxl @ file:///home/conda/feedstock_root/build_artifacts/openpyxl_1615404969632/work                                                                          packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1612459636436/work                              pandas @ file:///D:/bld/pandas_1614698123791/work                                                                                                                                                           pandocfilters==1.4.2                                                                                                                                              parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1607618318316/work                                      partd==1.1.0                              pathlib2 @ file:///D:/bld/pathlib2_1610137079782/work                                                                                                             pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work                                                                    Pillow @ file:///D:/bld/pillow_1615244129446/work                                                                       pluggy @ file:///D:/bld/pluggy_1610318289634/work                                                                                                                                                           prettytable @ file:///home/conda/feedstock_root/build_artifacts/prettytable_1614725168556/work                                                                    progressbar2 @ file:///home/conda/feedstock_root/build_artifacts/progressbar2_1599661727525/work                        prometheus-client @ file:///home/conda/feedstock_root/build_artifacts/prometheus_client_1605543085815/work                                                                                                  prompt-toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1616432837031/work                                                              protobuf==3.15.6                                                                                                        py @ file:///home/conda/feedstock_root/build_artifacts/py_1607783655754/work                                                                                                                                pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1593275161868/work                                                                        pydicom @ file:///home/conda/feedstock_root/build_artifacts/pydicom_1607562682046/work                                  Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1615243893546/work                                                                                                                    pygpu==0.7.6                                                                                                            PyOpenGL==3.1.5                           pyOpenSSL @ file:///home/conda/feedstock_root/build_artifacts/pyopenssl_1608055815057/work                                                                        pyparsing==2.4.7                                                                                                        Pypubsub==4.0.3                           PyQt5==5.12.3                                                                                                                                                     PyQt5-sip==4.19.18                                                                                                                                                PyQtChart==5.12                                                                                                         PyQtWebEngine==5.12.1                     pyreadline @ file:///D:/bld/pyreadline_1611175297308/work                                                                                                         pyrsistent @ file:///D:/bld/pyrsistent_1610146997928/work                                                               PySocks @ file:///D:/bld/pysocks_1610291658292/work                                                                                                                                                         pytest==6.2.2                                                                                                                                                     pytest-cov @ file:///home/conda/feedstock_root/build_artifacts/pytest-cov_1611167431813/work                            python-dateutil==2.8.1                    python-utils @ file:///home/conda/feedstock_root/build_artifacts/python-utils_1616867218381/work                                                                  pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1612179539967/work                                        PyWavelets @ file:///D:/bld/pywavelets_1607290963729/work                                                                                                                                                   pywin32==227                                                                                                            pywinpty @ file:///D:/bld/pywinpty_1602377770565/work                                                                                                                                                       PyYAML==5.4.1                                                                                                           pyzmq @ file:///D:/bld/pyzmq_1614611876813/work                                                                                                                                                             qtconsole @ file:///home/conda/feedstock_root/build_artifacts/qtconsole_1615946127199/work                                                                        QtPy==1.9.0                                                                                                             raven==6.10.0                             requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1608156231189/work                                                                          Rtree @ file:///D:/bld/rtree_1610147304955/work                                                                         scikit-image==0.14.2                      scikit-learn==0.19.2                                                                                                                                              scipy @ file:///C:/bld/scipy_1616878817947/work                                                                                                                   Send2Trash==1.5.0                                                                                                       Shapely==1.7.1                            six @ file:///home/conda/feedstock_root/build_artifacts/six_1590081179328/work                                                                                    snowballstemmer @ file:///home/conda/feedstock_root/build_artifacts/snowballstemmer_1611270869511/work                                                            Sphinx @ file:///home/conda/feedstock_root/build_artifacts/sphinx_1616256380770/work                                                                              sphinx-rtd-theme @ file:///home/conda/feedstock_root/build_artifacts/sphinx_rtd_theme_1609854399947/work                                                          sphinxcontrib-applehelp==1.0.2                                                                                                                                    sphinxcontrib-devhelp==1.0.2                                                                                                                                      sphinxcontrib-htmlhelp==1.0.3                                                                                                                                     sphinxcontrib-jsmath==1.0.1                                                                                                                                       sphinxcontrib-qthelp==1.0.3                                                                                                                                       sphinxcontrib-serializinghtml==1.1.4                                                                                                                              tabulate @ file:///home/conda/feedstock_root/build_artifacts/tabulate_1614001031686/work                                                                          tensorboard==1.13.1                                                                                                                                               tensorflow==1.13.1                                                                                                                                                tensorflow-estimator==1.13.0                                                                                                                                      termcolor==1.1.0                                                                                                                                                  terminado @ file:///D:/bld/terminado_1617048806218/work                                                                                                           testpath==0.4.4                                                                                                                                                   Theano @ file:///D:/bld/theano_1604228796430/work                                                                                                                 toml @ file:///home/conda/feedstock_root/build_artifacts/toml_1604308577558/work                                                                                  toolz @ file:///home/conda/feedstock_root/build_artifacts/toolz_1600973991856/work                                                                                tornado @ file:///D:/bld/tornado_1610094878337/work                                                                                                               tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1614982742480/work                                                                                  traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1602771532708/work                                                                        trimesh @ file:///home/conda/feedstock_root/build_artifacts/trimesh_1614383906608/work                                                                            typing-extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1602702424206/work                                                        urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1615828766818/work                                                                            wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1600965781394/work                                                                            webencodings==0.5.1                                                                                                                                               Werkzeug==1.0.1                                                                                                                                                   widgetsnbextension @ file:///D:/bld/widgetsnbextension_1605475785107/work                                                                                         win-inet-pton @ file:///D:/bld/win_inet_pton_1610147627696/work                                                                                                   wincertstore==0.2                                                                                                                                                 wxnatpy==0.3.2                                                                                                                                                    wxPython==4.1.1                                                                                                                                                   xnat @ file:///home/conda/feedstock_root/build_artifacts/xnat_1601289104072/work                                                                                  zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1614945704755/work

**Reply by MicPhotonIGBMC** [reply]
conda list
OUTPUT =
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: win-64
absl-py=0.12.0=pyhd8ed1ab_0
alabaster=0.7.12=py_0
albumentations=0.3.0=py_0
argon2-cffi=20.1.0=py37hcc03f2d_2
astor=0.8.1=pyh9f0ad1d_0
async_generator=1.10=py_0
atomicwrites=1.4.0=pyh9f0ad1d_0
attrs=20.3.0=pyhd3deb0d_0
axondeepseg=3.2.0=dev_0
babel=2.9.0=pyhd3deb0d_0
backcall=0.2.0=pyh9f0ad1d_0
backports=1.0=py_2
backports.functools_lru_cache=1.6.3=pyhd8ed1ab_0
blas=1.0=mkl
bleach=3.3.0=pyh44b312d_0
brotlipy=0.7.0=py37hcc03f2d_1001
ca-certificates=2021.1.19=haa95532_1
certifi=2020.12.5=py37haa95532_0
cffi=1.14.5=py37hd8e9650_0
chardet=4.0.0=py37h03978a9_1
cloudpickle=1.6.0=py_0
colorama=0.4.4=pyh9f0ad1d_0
contextlib2=0.6.0.post1=py_0
coverage=5.5=py37hcc03f2d_0
cryptography=3.4.7=py37h20c650d_0
cycler=0.10.0=py_2
cytoolz=0.11.0=py37hcc03f2d_3
dask-core=2021.3.1=pyhd8ed1ab_0
dataclasses=0.8=pyhc8e2a94_1
dcm2niix=1.0.20181114=h1ad3211_1000
decorator=4.4.2=py_0
defusedxml=0.7.1=pyhd8ed1ab_0
docutils=0.16=py37h03978a9_3
entrypoints=0.3=pyhd8ed1ab_1003
et_xmlfile=1.0.1=py_1001
freeglut=3.2.1=h0e60522_2
freetype=2.10.4=h546665d_1
fsleyes=0.33.1=py37hc8dfbb8_0
fsleyes-props=1.7.2=pyhd8ed1ab_0
fsleyes-widgets=0.11.2=pyhd8ed1ab_0
fslpy=3.5.3=pyh050c7b8_0
fsspec=0.8.7=pyhd8ed1ab_0
gast=0.4.0=pyh9f0ad1d_0
geos=3.9.1=h39d44d4_2
grpcio=1.36.1=py37h04d2302_0
h5py=2.10.0=nompi_py37h23cfb99_105
hdf5=1.10.6=nompi_h5268f04_1114
icu=68.1=h0e60522_0
idna=2.10=pyh9f0ad1d_0
imageio=2.9.0=py_0
imagesize=1.2.0=py_0
imgaug=0.4.0=py_1
importlib-metadata=3.9.1=py37h03978a9_0
importlib_metadata=3.9.1=hd8ed1ab_0
indexed_gzip=1.5.3=py37h5006e62_0
iniconfig=1.1.1=pyh9f0ad1d_0
intel-openmp=2020.3=h57928b3_311
ipykernel=5.5.0=py37heaed05f_1
ipython=7.22.0=py37h7813e69_0
ipython_genutils=0.2.0=py_1
ipywidgets=7.6.3=pyhd3deb0d_0
isodate=0.6.0=py_1
jasper=2.0.14=h77af90b_2
jdcal=1.4.1=py_0
jedi=0.18.0=py37h03978a9_2
jinja2=2.11.3=pyh44b312d_0
jpeg=9d=h8ffe710_0
jsonschema=3.2.0=pyhd8ed1ab_3
jupyter=1.0.0=py37h03978a9_6
jupyter_client=6.1.12=pyhd8ed1ab_0
jupyter_console=6.4.0=pyhd8ed1ab_0
jupyter_core=4.7.1=py37h03978a9_0
jupyterlab_pygments=0.1.2=pyh9f0ad1d_0
jupyterlab_widgets=1.0.0=pyhd8ed1ab_1
keras=2.3.1=py37h21ff451_0
keras-applications=1.0.8=py_1
keras-preprocessing=1.1.2=pyhd8ed1ab_0
kiwisolver=1.3.1=py37h8c56517_1
krb5=1.17.2=hbae68bd_0
lcms2=2.12=h2a16943_0
libblas=3.9.0=1_h8933c1f_netlib
libcblas=3.9.0=3_hd5c7e75_netlib
libclang=11.1.0=default_h5c34c98_0
libcurl=7.75.0=hf1763fc_0
libgpuarray=0.7.6=h8ffe710_1003
liblapack=3.9.0=3_hd5c7e75_netlib
liblapacke=3.9.0=3_hd5c7e75_netlib
libopencv=4.5.1=py37ha0199f4_0
libpng=1.6.37=h1d00b33_2
libprotobuf=3.15.6=h7755175_0
libsodium=1.0.18=h8d14728_1
libspatialindex=1.9.3=h39d44d4_3
libssh2=1.9.0=h680486a_6
libtiff=4.2.0=hc10be44_0
libwebp-base=1.2.0=h8ffe710_2
locket=0.2.0=py_2
lz4-c=1.9.3=h8ffe710_0
m2w64-gcc-libgfortran=5.3.0=6
m2w64-gcc-libs=5.3.0=7
m2w64-gcc-libs-core=5.3.0=7
m2w64-gmp=6.1.0=2
m2w64-libwinpthread-git=5.0.0.4634.697f757=2
mako=1.1.4=pyh44b312d_0
markdown=3.3.4=pyhd8ed1ab_0
markupsafe=1.1.1=py37hcc03f2d_3
matplotlib=3.3.4=py37h03978a9_0
matplotlib-base=3.3.4=py37h3379fd5_0
mistune=0.8.4=py37hcc03f2d_1003
mkl=2018.0.3=1
more-itertools=8.7.0=pyhd8ed1ab_0
mpld3=0.5.2=pyhd3deb0d_0
msys2-conda-epoch=20160418=1
nbclient=0.5.3=pyhd8ed1ab_0
nbconvert=6.0.7=py37h03978a9_3
nbformat=5.1.2=pyhd8ed1ab_1
nest-asyncio=1.4.3=pyhd8ed1ab_0
networkx=2.5=py_0
nibabel=3.2.1=pyhd8ed1ab_0
notebook=6.3.0=py37h03978a9_0
numpy=1.20.2=py37hcbcd69c_0
olefile=0.46=pyh9f0ad1d_1
opencv=4.5.1=py37h03978a9_0
opencv-contrib-python=4.5.1.48=pypi_0
opencv-python-headless=4.5.1.48=pypi_0
openpyxl=3.0.7=pyhd8ed1ab_0
openssl=1.1.1k=h2bbff1b_0
packaging=20.9=pyh44b312d_0
pandas=1.2.3=py37h08fd248_0
pandoc=2.13=h8ffe710_0
pandocfilters=1.4.2=py_1
parso=0.8.1=pyhd8ed1ab_0
partd=1.1.0=py_0
pathlib2=2.3.5=py37h03978a9_3
pickleshare=0.7.5=py_1003
pillow=8.1.2=py37h96663a1_0
pip=21.0.1=pyhd8ed1ab_0
pluggy=0.13.1=py37h03978a9_4
prettytable=2.1.0=pyhd8ed1ab_0
progressbar2=3.53.1=pyh9f0ad1d_0
prometheus_client=0.9.0=pyhd3deb0d_0
prompt-toolkit=3.0.18=pyha770c72_0
prompt_toolkit=3.0.18=hd8ed1ab_0
protobuf=3.15.6=py37hf2a7229_0
pthread-stubs=0.4=hcd874cb_1001
py=1.10.0=pyhd3deb0d_0
py-opencv=4.5.1=py37heaed05f_0
pycparser=2.20=pyh9f0ad1d_2
pydicom=2.1.2=pyhd3deb0d_0
pygments=2.8.1=pyhd8ed1ab_0
pygpu=0.7.6=py37hda49f71_1002
pyopengl=3.1.5=py_0
pyopenssl=20.0.1=pyhd8ed1ab_0
pyparsing=2.4.7=pyh9f0ad1d_0
pypubsub=4.0.3=py_0
pyqt=5.12.3=py37h03978a9_7
pyqt-impl=5.12.3=py37hf2a7229_7
pyqt5-sip=4.19.18=py37hf2a7229_7
pyqtchart=5.12=py37hf2a7229_7
pyqtwebengine=5.12.1=py37hf2a7229_7
pyreadline=2.1=py37h03978a9_1003
pyrsistent=0.17.3=py37hcc03f2d_2
pysocks=1.7.1=py37h03978a9_3
pytest=6.2.2=py37h03978a9_0
pytest-cov=2.11.1=pyh44b312d_0
python=3.7.10=h7840368_100_cpython
python-dateutil=2.8.1=py_0
python-utils=2.5.6=pyh44b312d_0
python_abi=3.7=1_cp37m
pytz=2021.1=pyhd8ed1ab_0
pywavelets=1.1.1=py37hda49f71_3
pywin32=227=py37he774522_1
pywinpty=0.5.7=py37hc8dfbb8_1
pyyaml=5.4.1=py37hcc03f2d_0
pyzmq=22.0.3=py37hcce574b_1
qt=5.12.9=h5909a2a_4
qtconsole=5.0.3=pyhd8ed1ab_0
qtpy=1.9.0=py_0
raven=6.10.0=py_0
requests=2.25.1=pyhd3deb0d_0
rtree=0.9.7=py37h13cc57e_1
scikit-image=0.14.2=py37h6538335_1
scikit-learn=0.19.2=py37heebcf9a_0
scipy=1.6.2=py37h924764e_0
send2trash=1.5.0=py_0
setuptools=49.6.0=py37h03978a9_3
shapely=1.7.1=py37hdcc0ea0_4
six=1.15.0=pyh9f0ad1d_0
snowballstemmer=2.1.0=pyhd8ed1ab_0
sphinx=3.5.3=pyhd8ed1ab_0
sphinx_rtd_theme=0.5.1=pyhd3deb0d_0
sphinxcontrib-applehelp=1.0.2=py_0
sphinxcontrib-devhelp=1.0.2=py_0
sphinxcontrib-htmlhelp=1.0.3=py_0
sphinxcontrib-jsmath=1.0.1=py_0
sphinxcontrib-qthelp=1.0.3=py_0
sphinxcontrib-serializinghtml=1.1.4=py_0
sqlite=3.35.3=h8ffe710_0
tabulate=0.8.9=pyhd8ed1ab_0
tensorboard=1.13.1=py37_0
tensorflow=1.13.1=h21ff451_7
tensorflow-base=1.13.1=py37_7
tensorflow-estimator=1.13.0=py37h39e3cac_0
termcolor=1.1.0=py_2
terminado=0.9.4=py37h03978a9_0
testpath=0.4.4=py_0
theano=1.0.5=py37hf2a7229_1
tk=8.6.10=h8ffe710_1
toml=0.10.2=pyhd8ed1ab_0
toolz=0.11.1=py_0
tornado=6.1=py37hcc03f2d_1
tqdm=4.59.0=pyhd8ed1ab_0
traitlets=5.0.5=py_0
trimesh=3.9.8=pyh44b312d_0
typing_extensions=3.7.4.3=py_0
urllib3=1.26.4=pyhd8ed1ab_0
vc=14.2=hb210afc_4
vs2015_runtime=14.28.29325=h5e1d092_4
vs2017_win-64=19.16.27038=h2e3bad8_2
vswhere=2.8.4=h57928b3_0
wcwidth=0.2.5=pyh9f0ad1d_2
webencodings=0.5.1=py_1
werkzeug=1.0.1=pyh9f0ad1d_0
wheel=0.36.2=pyhd3deb0d_0
widgetsnbextension=3.5.1=py37h03978a9_4
win_inet_pton=1.1.0=py37h03978a9_2
wincertstore=0.2=py37h03978a9_1006
winpty=0.4.3=4
wxnatpy=0.3.2=py_0
wxpython=4.1.1=py37ha2e4c6b_1
xnat=0.3.25=pyh9f0ad1d_0
xz=5.2.5=h62dcd97_1
yaml=0.2.5=he774522_0
zeromq=4.3.4=h0e60522_0
zipp=3.4.1=pyhd8ed1ab_0
zlib=1.2.11=h62dcd97_1010
zstd=1.4.9=h6255e5f_0

**Reply by mathieuboudreau** [reply]
Perfect, thank you! @bvernay @MicPhotonIGBMC can you confirm that you are trying to run FSLeyes on a computer running a Windows OS?

**Reply by MicPhotonIGBMC** [reply]
anaconda 4.9.2, Windows 10, non admin account

**Reply by mathieuboudreau** [reply]
Perfect - thanks for clarifying @MicPhotonIGBMC !

Unfortunately, the external tool that we use for a GUI (FSLeyes) is not currently compatible with Windows - we mention it in a `Note` in our installation instructions [here](https://axondeepseg.readthedocs.io/en/latest/documentation.html#install-axondeepseg). 

* All of their (FSLeyes) documentation is only for Mac/Linux: https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLeyes and https://users.fmrib.ox.ac.uk/~paulmc/fsleyes/userdoc/latest/install.html
* It's also mentioned by the developer in some GitHub issues, such as this one: https://github.com/pauldmccarthy/fsleyes/issues/24#issuecomment-498588513

A workaround would be to install FSLeyes/AxonDeepSeg within a Virtual Machine (e.g. VirtualBox) running a Linux instance. We could guide you with that if you'd like; I think @Stoyan-I-A or @vs74 have done it before?

**Reply by Stoyan-I-A** [reply]
Hello @bvernay @MicPhotonIGBMC ,

Both me and @mariehbourget use a Windows 10 Machine. However, since FSLeyes is not supported on Windows, we have to use some workarounds to run it.

@mariehbourget uses the [Linux Subsystem for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10).

I myself use a Virtual Machine (VM) to run a Linux "virtual installation" on my machine. It works well but it has a few hardware requirements:
1- It requires a lot of available space on your Disk storage (think around 50 Gb to be safe) since you're essentially installing another operating system.
2- The Virtual Machine requires you to allocate a lot of RAM to it. I can't give you a specific number, but with 8 Gb allocated to it I didn't have any memory issues. So if your Windows machine has 16 Gb of RAM, you should be fine. I was able to run the Virtual Machine on a laptop with around 5 Gb allocated to the VM, but it had trouble processing some images if I had too many other apps open.

To use a virtual Linux installation, you first need to install a Virtual Machine software (I use [VMware](https://www.vmware.com/ca-fr/products/workstation-player.html) but you can also use [virtual box](https://www.virtualbox.org/) ).
Second, you need to download a Linux .iso file. This is required by the Virtual Machine software to create the Linux operating system. I use [Ubuntu 20.04](https://ubuntu.com/download/desktop).

This youtube video can help you setup the Virtual Linux environment on VMware: https://www.youtube.com/watch?v=9rUhGWijf9U
Here's an equivalent tutorial for VirutalBox: https://www.youtube.com/watch?v=x5MhydijWmc

Once your virtal Linux installation is ready, you can follow the instructions on our [documentation ](https://axondeepseg.readthedocs.io/en/latest/documentation.html)for the installing AxonDeepSeg on Linux.

**Reply by MicPhotonIGBMC** [reply]
Thanks very much for the details instructions. I'll try when I find the time since I'm very happy working with the notebooks. Thanks again for the tool and the support.

Bertrand
